Community contributions documentation

quadrophobiac opened the pull request:
I'm moving the draft text from ODI Contributor Guide: 2nd Draft into the README of this repo.
I'd like some feedback on how this reads - is there anything I've missed and should include? @JeniT @Floppy @ldodds @davetaz @pikesley @pezholio

quadrophobiac committed:
Including toolbox contributors guide as README

quadrophobiac committed:
Include code of conduct

quadrophobiac committed:
Changes because of different readme in this repo

quadrophobiac committed:
Hyperlink resolution

quadrophobiac committed:
Include link to push access heading

Stephen-Gates commented on README.md:
```diff
@@ -0,0 +1,146 @@
+ 
+The Toolbox grew out of ODI Labs engagement with the open data software ecosystem. Noticing gaps in the types of data tools available Labs developed MVP experimental products intended to plug those gaps. We hope these provocations get everyone thinking. We’d like for the community working in open data software to either fill the identified gaps in other ways, use our work to continue filling thos ...
+ 
+We look to OSS success stories like Ruby, Github and Adapt and we want to build an ecosystem of OSS tools that support Open Data - like OKFN’s JKAN. We know we can’t do this all by ourselves, nor would we see this as the best way forward - we aspire to build with developments in the ecosystem. We want to build an OSS community around the Toolbox and hope you can be part of it!
```
Do you mean Open Knowledge International's CKAN project?

Stephen-Gates commented on README.md:
```diff
@@ -0,0 +1,146 @@
+
+## Contributing Code
+
+If you’re a developer interested in Open Data familiar with any of the following technologies we’d like to interest you in collaborating on our Toolbox
```
If you’re a developer interested in Open Data **and** familiar with any of the following technologies **,** we’d like to interest you in collaborating on our Toolbox

quadrophobiac committed:
Including copy edit suggestions

quadrophobiac committed:
Tidy up toolbox integration example

quadrophobiac committed:
Remove organisation centric introduction

quadrophobiac committed:
Hyperlinks and MD formatting amendments

quadrophobiac commented:
Thanks for the feedback on this - I have parcelled out the contents of this readme to a wiki within this repository in case you have any further suggestions for improvement https://github.com/theodi/toolbox/wiki

quadrophobiac committed:
Include labs flow image for cref elsewhere

quadrophobiac committed:
Merge branch 'master' into community-contributions-documentation

Resolve Gitter badge conflict

quadrophobiac committed:
include Gitter channel in resources for dev

quadrophobiac commented:
Expanding scope of this PR to include image assets for use in Wiki

quadrophobiac committed:
Image for illustrating label hierarchy

quadrophobiac committed:
Include issue ticket template inclusion

quadrophobiac committed:
Amendments to this readme, reflecting status of rep

This repo is a placeholder for the Toolbox in general so amendments to
copy required

quadrophobiac committed:
Including label screenshots

quadrophobiac committed:
Amend README

include information apt for the actual readme of this repository

quadrophobiac commented:
additions made, not showing up here
